packages/apidom-converter/src/strategies/openapi-3-1-to-openapi-3-0-3/refractor-plugins/security-scheme-type.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  ComponentsElement,
+  OpenApi3_1Element,
+  SecurityRequirementElement,
+  SecuritySchemeElement,
+} from '@swagger-api/apidom-ns-openapi-3-1';
+import {
+  AnnotationElement,
+  toValue,
+  cloneShallow,
+  isObjectElement,
+} from '@swagger-api/apidom-core';
+import { isComponentsElement } from '@swagger-api/apidom-ns-openapi-3-0';
+
+type SecuritySchemeTypePluginOptions = {
+  annotations: AnnotationElement[];
+};
+
+const isSecuritySchemeElement = (element: SecuritySchemeElement) => !!element; // TODO: Add import from @swagger-api/apidom-ns-openapi-3-1 when new release is released
```

Comment:
I've just added https://github.com/swagger-api/apidom/pull/3811, which adds `isSecuritySchemeElement` to the OpenAPI 3.1.0 namespace (which is what we really need here and not OpenAPI 3.0 predicate).

```js
import { isComponentsElement, isSecuritySchemeElement } from '@swagger-api/apidom-ns-openapi-3-1';
```

This will work. You don't need to wait for the release. Pull the latest changes from `main`, merge it to this branch, run `npm run build:cjs` && `npm run typescript:declaration` from the top of the monorepo, and the code in this PR will automatically start seeing the predicate.

Reply:
Thanks, this worked. I was aware of the build step and i did that but indeed `npm run typescript:declaration` was needed too.